mapping.md

```diff
@@ -68,21 +68,15 @@ Maps to nix.Source with `type = neo.recordingchannelgroup`.
     | RecordingChannelGroup.name(string)        | Source.name(string)                   |
     | RecordingChannelGroup.description(string) | Source.definition(string)             |
     | RecordingChannelGroup.file_origin(string) | Source.metadata(**Section**) [[1]](#notes) |
-    | RecordingChannelGroup.coordinates(Quantity 2D) | Source.metadata(**Section**) [[1]](#notes) |
 
+  - Objects
+    - For each channel in `RecordingChannelGroup`, determined by the `channel_indexes` list, a `neo.Unit` is created with `type = neo.recordingchannel`.
```

Comment:
Why has the `neo.Unit` the type `neo.recordingchannel` and why is a unit created for each channel index?